platforms/documentation/docs/src/docs/userguide/reference/runtime-configuration/gradle_wrapper.adoc

```diff
@@ -317,7 +317,7 @@ Check out the link:{javadocPath}/org/gradle/api/tasks/wrapper/Wrapper.html[API d
 [[sec:authenticated_download]]
 === Authenticated Gradle distribution download
 
-The Gradle `Wrapper` can download Gradle distributions from servers using HTTP Basic Authentication.
+The Gradle `Wrapper` can download Gradle distributions from servers using HTTP Basic Authentication or with a static HTTP Bearer Token.
```

Comment:
There is a section below stating that Basic Authentication should only be used with HTTPS. That's just as important for bearer tokens too, so that section should be updated too.